tests/e2e/specs/wcpay/shopper/shopper-checkout-purchase-site-editor.spec.ts

```diff
@@ -27,71 +27,85 @@ import {
  * Tests for successful purchases with both card testing prevention enabled
  * and disabled states using a site builder enabled theme.
  */
-[ true, false ].forEach( ( cardTestingPreventionEnabled ) => {
```

Comment:
Slightly optimized this test (~10% faster), removing the external `.forEach`, so that the theme change doesn't occur as often.

Before:
<img width="585" alt="Screenshot 2025-04-17 at 11 06 57 AM" src="https://github.com/user-attachments/assets/86cc636c-6dd9-4925-887e-796aa1cbaab1" />

After:
<img width="593" alt="Screenshot 2025-04-17 at 11 01 50 AM" src="https://github.com/user-attachments/assets/8a6beb33-a9b0-429a-a47e-8345a75977db" />